lib/openxml/docx/style.rb

```diff
@@ -84,11 +84,19 @@ def install_table_properties
       end
 
       def property_xml(xml)
+        style_property_xml(xml)
+
         return table.property_xml(xml) if table_style?
         character.property_xml(xml)
         paragraph.property_xml(xml) if paragraph_style?
       end
 
+      def style_property_xml(xml)
+        props = properties.keys.map(&method(:send)).compact
```

Comment:
I don't believe there's another way than a `send`. I pulled the core of this method implementation from `property_builder.rb`, so if there's a security vulnerability, we should definitely refactor in both places.